ElectronicObserver/Window/Wpf/Fleet/ViewModels/FleetItemViewModel.cs

```diff
@@ -426,6 +435,38 @@ private string GetEquipmentString(IShipData ship)
 		EngagementType engagement = (EngagementType)Utility.Configuration.Config.Control.PowerEngagementForm;
 		IFleetData fleet = KCDatabase.Instance.Fleet[Parent.FleetId];
 
+		if (SpecialAttackHitList.Any())
+		{
+			sb.AppendFormat($"\r\nSpecial attacks (Day) :\r\n");
+
+			foreach (KeyValuePair<SpecialAttack, List<SpecialAttackHit>> specialAttackData in SpecialAttackHitList)
```

Comment:
You should be able to just deconstruct the KeyValuePair.
```cs
foreach ((SpecialAttack attack, List<SpecialAttackHit> hits) in SpecialAttackHitList)
```

Reply:
Should be good now